src/dmd/cppmangle.d

```diff
@@ -162,19 +162,112 @@ private final class CppMangleVisitor : Visitor
                 !getQualifier(s));      // at global level
     }
 
+    /******************************
+     * Write the mangled representation of a template argument.
+     * Params:
+     *  ti  = the template instance
+     *  arg = the template argument index
+     */
+    void template_arg(TemplateInstance ti, size_t arg = 0)
```

Comment:
Try to avoid default arguments, then it will make it easier to get the PR passed Walter.